content/docs/references/contributing.md

```diff
@@ -92,3 +92,9 @@ Ask the following questions to yourself whenever you find that you are unsure wh
 ### Need for judicial application
 
 It is important to reiterate that the decision will involve discrete judgment and possibly, discussion. Use the above points and the flowchart as a reference, but in the end, apply your judgment. We are flexible in changing our stance wherever it makes sense. Create an issue, open a PR, or just send a message on Slack to talk about your content and where it can go in the engineering handbook.
+
+### Markdown: Our Choice for Documentation
+
+The Engineering Handbook uses [Markdown](https://www.markdownguide.org/basic-syntax/) primarily due to its target audience: engineers. Markdown, a developer-friendly format, simplifies the editing process and integrates seamlessly with version control systems like Git. Currently, the handbook is hosted on GitHub as a public repository, reflecting our commitment to openness and engineering proficiency. This decision has two implications: pragmatically, it’s easier for developers to work with, and aspirationally, it encourages engineers to regularly interact with command-line and coding-related tools outside their projects, reinforcing our engineering-focused culture.
+
+When contributing to the Engineering Handbook using Markdown, it’s important to be mindful of special texts that may be responsible for formatting or creating dynamic content. For instance, you might come across a section at the top enclosed between `---` lines. This is known as the “front matter”, and it’s where you can set predefined page variables. There are also shortcodes that are used to create dynamic content within your Markdown files. They are specific to the system you’re using and are typically enclosed in double curly braces `{{ }}`. For example, `{{< section-pages >}}` is a shortcode that dynamically generates a list of all pages in the current section.
```

Comment:
`{{< section-pages >}}` is not rendered correctly. See https://deploy-preview-121--axelerant-engg-handbook.netlify.app/docs/references/contributing/#markdown-our-choice-for-documentation

I don't know the fix off-hand. I suggest trying this locally to see what variant fixes it.

Reply:
For the moment, I am able to display the shortcode without the code format.